qa/paqa_macros.h

```diff
@@ -0,0 +1,74 @@
+
+#ifndef PORTAUDIO_QA_PAQA_TOOLS_H
+#define PORTAUDIO_QA_PAQA_TOOLS_H
+
+static int gNumPassed = 0; /* Two globals */
+static int gNumFailed = 0;
```

Comment:
These really are "globals". Declaring them static in a header file seems risky to me -- what happens if there is more than one `.c` file in the test program? 

I would prefer to make them real globals: declare them here, and define a macro that can be used to instantiate them in the compilation unit that contains main. E.g.

```
In this .h:

extern int gNumPassed;
extern int gNumFailed;

/* you must use this macro exactly once in each test program */
#define PAQA_INSTANTIATE_GLOBALS\
    int gNumPassed = 0;\
    int gNumPassed = 0;

In the .c file containing `main()`:

PAQA_INSTANTIATE_GLOBALS
```

maybe also rename `gpaqaNumPassed` or something more consistent with our naming scheme.